docs/changehistory/NextVersion.md

```diff
@@ -268,6 +268,9 @@ SAML support has officially been dropped as a supported workflow. All related AP
 | `ReactMessage`                         | `ReactMessage` in @bentley/ui-core                                                     |
 | `SpecialKey`                           | `SpecialKey` in @bentley/ui-abstract                                                   |
 | `WidgetState`                          | `WidgetState` in @bentley/ui-abstract                                                  |
+| `UserProfileBackstageItem`             | *eliminated*                                                                           |
```

Comment:
What should an app that was using these eliminated APIs use instead?

Reply:
I *think* the plan is for users to implement their own SignIn/Signout components using the AuthenticationClient. @calebmshafer can confirm/deny.

Reply:
Yes, the plan is to have them directly implement a sign-in and sign-out component. The iTwin Viewer doesn't currently use this, nor do most apps other than test apps, because they all have different ways of handling sign-in prior to even loading the viewer.

Reply:
We can definitely provide examples and I think the Viewer intends on coming with a default set of components for it.

Reply:
Idk if the viewer will have a default set of components for it, maybe, but seems unnecessary if its just going to be a button that just calls Sign in which will redirect to bentley idp or a sign out. Will look into it, but i think its fine to completely remove these from core altogether